Src/Managers/size_manager.py

```
@@ -0,0 +1,102 @@
+                (theme, item_type.mvName) in processed_themes: continue
+            if Themes.RESIZABLE not in theme: continue
+            if not (component := self.__theme_manager \
+                .get_component(*theme, component=item_type)): # Возможно стоит добавить кэширование на методы theme_manager
+                    continue
+
+            default_elements = self.__theme_manager.config \
+                            .get(Themes.RESIZABLE.value) \
+                            .get(item_type.mvName, {})
+
+            for element in get_children(component, depth=1):
+                if (element := dpg.get_item_alias(element)).split()[2] not in default_elements: continue
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)